Gemfile

```diff
@@ -14,6 +14,9 @@ group :development, :test do
   gem 'simplecov'
 end
 
+# Travis CI
```

Comment:
You should edit your comment. Rake is not exclusive to Travis CI.

Also make sure to run `bundle install` after editing Gemfile, so Gemfile.lock gets updated as well.